libs/common-const/src/bff.ts

```diff
@@ -1 +1 @@
-export const BFF_BASE_URL = process.env.REACT_APP_BFF_BASE_URL || 'https://bff.cow.fi'
\ No newline at end of file
+export const BFF_BASE_URL = process.env.REACT_APP_BFF_BASE_URL || 'https://bff.barn.cow.fi'
```

Comment:
Should it default to barn?
Need to double check the env var settings before deploying this to be safe.

Reply:
This env var is not set by default only in localhost.
Traditionally, in localhost we use Barn environment